paper-swatch-picker.js

```diff
@@ -18,39 +30,26 @@ Example:
     <paper-swatch-picker color="{{selectedColor}}"></paper-swatch-picker>
 
 You can configure the color palette being used using the `colorList` array and
-the `columnCount` property, which specifies how many "generic" colours (i.e. columns
-in the picker) you want to display.
+the `columnCount` property, which specifies how many "generic" colours (i.e.
+columns in the picker) you want to display.
 
-    <paper-swatch-picker column-count=5 color-list='["#65a5f2", "#83be54", "#f0d551", "#e5943c", "#a96ddb"]'></paper-swatch-picker>
+    <paper-swatch-picker column-count=5 color-list='["#65a5f2", "#83be54",
+"#f0d551", "#e5943c", "#a96ddb"]'></paper-swatch-picker>
 
 ### Styling
 
 The following custom properties and mixins are available for styling:
 
 Custom property | Description | Default
 ----------------|-------------|----------
-`--paper-swatch-picker-color-size` | The size of each of the color boxes | `20px`
+`--paper-swatch-picker-color-size` | The size of each of the color boxes |
+`20px`
```

Comment:
This table was broken by the formatter - you can prevent webmat from changing comments with [this](https://github.com/PolymerElements/iron-fit-behavior/pull/93#discussion_r199271183).